parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -401,17 +402,20 @@ private static boolean isMinMaxStatsSupported(PrimitiveType type) {
   static org.apache.parquet.column.statistics.Statistics fromParquetStatisticsInternal
       (String createdBy, Statistics formatStats, PrimitiveType type, SortOrder typeSortOrder) {
     // create stats object based on the column type
-    org.apache.parquet.column.statistics.Statistics stats = org.apache.parquet.column.statistics.Statistics.createStats(type);
+    StatisticsBuilder builder = org.apache.parquet.column.statistics.Statistics.getBuilder(type);
```

Comment:
(optional, nit): I think `stats` would be more descriptive regarding the purpose of this variable, even if it is a builder. Alternatively, `statsBuilder`.